### DISCUSSION TITLE
Packages Roadmap Cut?

### DISCUSSION THREAD

**Discussion by RobbieMcKinstry** [community/community#29579] [Actions] [answered]
Hello!

Earlier this year, I read on the [GitHub Public Roadmap](https://github.com/orgs/github/projects/4247/views/1) GitHub Packages were expecting several new features. For example, I recall Helm support slated for Q3, among other languages like Python and PHP.

However, now I see no new features are planned, and the tickets for those languages have been moved into the "future" column. It would appear that GitHub is shifting engineering focus away from Packages to other areas of the organization. Should I expect this to be a trend?

**Comment by tinaheidinger** [comment, marked as answer]
Hi, Packages PM here 👋 
I can assure you that there is no focus shift away from Packages. In fact, we'll be updating our public roadmap more actively. Stay tuned for more transparency on what we're committed to, including improved capabilities and new ecosystems.
If there's anything specific you're looking forward to, please let us know by submitting feature requests!

**Comment by RobbieMcKinstry** [comment]
Hi Tina! Thank you very much for the reply; I'm thrilled to hear it. :-) Packages is such an integral product, especially because it dovetails so well with GitHub Actions. Am I correct that [this](https://github.com/community/community/discussions/categories/actions-and-packages) Discussions category is the best place to submit a feature request?

**Reply by tinaheidinger** [reply]
Yes that's right! :)
